src/routes/RepositoriesList/RepositoriesList.tsx

```diff
@@ -408,15 +437,14 @@ export default function RepositoriesList() {
                   </Td>
                   <Td dataLabel={columnNames.visibility}>
                     {' '}
-                    {repo.isPublic ? 'public' : 'private'}
+                    {repo.is_public ? 'public' : 'private'}
                   </Td>
-                  <Td dataLabel={columnNames.tags}> {repo.tags} </Td>
-                  <Td dataLabel={columnNames.size}> {repo.size} </Td>
-                  <Td dataLabel={columnNames.pulls}> {repo.pulls} </Td>
-                  <Td dataLabel={columnNames.lastPull}> {repo.lastPull} </Td>
+                  <Td dataLabel={columnNames.tags}> dummy </Td>
```

Comment:
Do these need to be dummy?

Reply:
replaced with `-` for now. These will be soon addressed as part of https://issues.redhat.com/browse/PROJQUAY-4082